apps/api/requirements-api.txt

```diff
@@ -8,3 +8,4 @@ requests
 serverless_wsgi>=1.7.8
 shapely
 strict-rfc3339
+-e lib/dynamo
```

Comment:
I don't actually see this folder/library in the PR, did you push it?

Reply:
I did not